Formula/octave.rb

```diff
@@ -96,6 +96,18 @@ def install
                           "--with-sndfile"
     system "make", "all"
 
+    on_linux do
```

Comment:
Do you always want to do this? Since we only have an `automake` dependency for `HEAD`. Or does it get pulled in as a recursive dep?

Reply:
Actually I forgot to add these dependencies. I did that, automake and autoconf have now been added.